app/components/sidebars/main/channels_list/list/list.js

```diff
@@ -81,6 +80,12 @@ export default class List extends PureComponent {
         });
     }
 
+    componentDidMount() {
+        if (!UnreadIndicator) {
+            UnreadIndicator = require('app/components/sidebars/main/channels_list/unread_indicator').default;
```

Comment:
Why do we have this conditional?

Reply:
Aah i see we had it before as conditional as well but was there a reason?

Reply:
so it is not included in the main bundle to improve startup time

Reply:
Ya, i thought so but it felt weird doing it for one file or component alone. i don't see why it was done for this particular component.

Reply:
cause this component is the only component here that does not necessarily need to be part of the same bundle as it does not show when the component mounts